### ISSUE TITLE
startSketchOn("XY") |> startProfileAt([0,0], %) doesn't show anything because there's no segments added yet, but this is confusing

### ISSUE THREAD

**Issue by nadr0** [KittyCAD/modeling-app#4126]
I debugged this and realized that the issue is the `default_camera_look_at` is not be triggered in a few scenarios. Not just this workflow. 

```
startSketchOn("XY") |> startProfileAt([0,0], %) 
```
This block of code will not triggered `default_camera_look_at`

```
{"success":true,"request_id":"ba618b70-b426-41de-bbb7-ba00ccc740e3","resp":{"type":"modeling","data":{"modeling_response":{"type":"zoom_to_fit","data":{"settings":{"pos":{"x":null,"y":null,"z":null},"center":{"x":0.0,"y":0.0,"z":0.0},"up":{"x":0.0,"y":0.0,"z":1.0},"orientation":{"x":-0.5380994,"y":0.0,"z":0.0,"w":0.8428814},"fov_y":45.0,"ortho_scale":null,"ortho":false}}}}}}
```

`zoom_to_fit` is failing as well. It is giving `null` for the position.

**Comment by franknoirot** [comment]
I agree with you @nadr0. Do you have any ideas what the fix should be? Here are some of mine:
1. Update the logic that determines if we should show the default planes to consider not just [if the artifactGraph is empty as it does now](https://github.com/KittyCAD/modeling-app/blob/main/src/lang/std/engineConnection.ts#L2192-L2196), but also if it contains elements that ought to be visible (not sure where this mapping of artifact types to "visibility" status would live)
2. Show a plane in the scene like how unused Offset Plane artifacts show in the scene
3. Both 1 and 2?

**Comment by franknoirot** [comment]
Closing as completed @nadr0. The feature tree helps show that a Sketch is present, and a point appears for me in the scene, indicating that a path has been started even without looking at the code pane:

![Showing a scene with a sketch only containing a `startProfileAt` with the feature tree pane open, with a dot marking the profile start visible in the scene](https://github.com/user-attachments/assets/3586315b-dfd1-4089-8fd2-5f55fb39ce91)
